opal/mca/btl/openib/btl_openib_component.c

```diff
@@ -2635,23 +2635,10 @@ btl_openib_component_init(int *num_btl_modules,
         goto no_btls;
     }
 
-    /* If we want fork support, try to enable it */
-#ifdef HAVE_IBV_FORK_INIT
-    if (0 != mca_btl_openib_component.want_fork_support) {
-        if (0 != ibv_fork_init()) {
-            /* If the want_fork_support MCA parameter is >0, then the
-               user was specifically asking for fork support and we
-               couldn't provide it.  So print an error and deactivate
-               this BTL. */
-            if (mca_btl_openib_component.want_fork_support > 0) {
-                opal_show_help("help-mpi-btl-openib.txt",
-                               "ibv_fork_init fail", true,
-                               opal_process_info.nodename);
-                goto no_btls;
-            }
-        }
+    /* If fork support is requested, try to enable it */
```

Comment:
Mmmm.  That's an interesting case (the fact that the MCA param param and its synonyms are registered at different times effectively creates different interpretations of what the value should be).  @hjelmn Got any suggestions here?

The important thing here is that there was definitely a time when the fork MCA param worked properly.  Perhaps it doesn't work now, but there was a window of time where it *did* work.  It would be worth finding out how large that window was to establish some sort of baseline of probability of how many users may actually be using it (and would be harmed if you silently deleted it).

More specifically: one way or another, the MCA var "ABI" issue here has to be addressed: you can't just silently delete a public MCA var; it has to be handled somehow.